FakeItEasy.Analyzers.sln

```diff
@@ -61,26 +61,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakeItEasy.Analyzer.VisualB
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tools", "tools", "{9D99B251-FB4F-49BF-8405-F6E0DE46EB32}"
 	ProjectSection(SolutionItems) = preProject
-		tools\deploy.cmd = tools\deploy.cmd
-		tools\Directory.Build.props = tools\Directory.Build.props
 		tools\FakeItEasy.ToolPaths.targets = tools\FakeItEasy.ToolPaths.targets
 		tools\prepare_release.cmd = tools\prepare_release.cmd
 	EndProjectSection
 EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "FakeItEasy.Analyzer.Shared", "src\FakeItEasy.Analyzer.Shared\FakeItEasy.Analyzer.Shared.shproj", "{E3ED4908-BC8D-4D81-A31B-8CD0B67F41B4}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakeItEasy.Build", "tools\FakeItEasy.Build\FakeItEasy.Build.csproj", "{81779103-4CA9-4177-B78A-A435FC37DADC}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakeItEasy.PrepareRelease", "tools-shared\FakeItEasy.PrepareRelease\FakeItEasy.PrepareRelease.csproj", "{0E2ED1CD-C7ED-4641-9482-C4BBBA58B623}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FakeItEasy.Deploy", "tools-shared\FakeItEasy.Deploy\FakeItEasy.Deploy.csproj", "{5F4FC5FA-E2FA-456D-813F-6057B7CE7FF0}"
-EndProject

```

Comment:
None of these exist.

Reply:
They should exist...
![image](https://user-images.githubusercontent.com/1864281/210960920-bc7b0406-6f47-4d2d-af63-4dbdaa1bdf49.png)
Did you restore the submodule?

Anyway, we probably don't need them in the solution...

Reply:
Yeah, that was my bad. I didn't restore the submodule until later, and forgot to go back.
I agree, though. No point adding the tools-shared items. Will keep out. And also remove FakeItEasy.ToolPaths.targets, which I missed before!